src/ifp/src/InitFloorplan.cc

```diff
@@ -407,6 +407,14 @@ void InitFloorplan::insertTiecells(odb::dbMTerm* tie_term,
 
   auto* port = network_->dbToSta(tie_term);
   auto* lib_port = network_->libertyPort(port);
+  if (!lib_port) {
+    logger_->error(utl::IFP,
+                   39,
+                   "liberty for this cell or the port is missing in the liberty {}/{}.",
```

Comment:
"in the liberty" seems redundant

No need for a return after this as it will throw an exception.